website/docs/guides/container_service_extension.html.markdown

```diff
@@ -0,0 +1,433 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Container Service Extension"
+sidebar_current: "docs-vcd-guides-cse"
+description: |-
+Provides guidance on configuring VCD to be able to install Container Service Extension.
+---
+
+# Container Service Extension
+
+## About
+
+This guide describes the required steps to configure VCD to install the Container Service Extension (CSE), that
+will allow tenant users to deploy Kubernetes clusters on VCD using the UI. For that purpose, after completing the steps described below you
+will need also to **publish the Container UI Plugin** to the desired tenants and **run the CSE server** in your infrastructure.
+
+To know more about CSE, you can explore [the official website](https://vmware.github.io/container-service-extension/).
+
+## Pre-requisites
+
+In order to complete the steps described in this guide, please be aware:
+
+* CSE is supported from VCD 10.3.1 or above, make sure your VCD appliance matches the criteria.
+* Terraform provider needs to be v3.7.0 or above.
+* All CSE elements use NSX-T backed resources, **no** NSX-V is supported.
+* Some steps require the usage of `cse` extension for `vcd-cli`. Make sure you have them installed. Also, `cse` server needs
+to be run when the steps are completed.
+
+## Installation process
```

Comment:
I think a hint to examples is also worth to be put here (I know it is at the bottom of the page), but for the impatient ones that could be nice to just jump to example.

Reply:
Done, thanks.